db/migrate/20191004151515_create_versioned_school_certifying_officials_archives.rb

```diff
@@ -1,4 +1,4 @@
-class CreateVersionedSchoolCertifyingOfficialsArchives < ActiveRecord::Migration[4.2]
+class CreateVersionedSchoolCertifyingOfficialsArchives < ActiveRecord::Migration[5.0]
```

Comment:
Did this not work with 4.2? I don't see a reason to change it but either is fine.

Reply:
Just cleanup, really.  I'm trying to be consistent and make these match the actual version when the migration was added.